src/customer-service/Actions/CreateIncidentAction.cs

```diff
@@ -0,0 +1,217 @@
+﻿using customer_service.Factories;
+using customer_service.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace customer_service.Actions
+{
+    public class CreateIncidentAction
+    {
+        /**
+         * Class: CreateIncidentAction
+         * Purpose: Prompts the user for information about a new incident and saves that incident to the database
+         * Author: Garrett Vangilder
+         * Methods:
+         *     static void ReadInput() - Prompts for incident info and saves to DB
+         */
+        public CreateIncidentAction()
+        {
+
+        }
+        /**
+        * Purpose: Reads user input for new incident information
+        * Return:
+        *     void
+        */
+        public static void ReadInput()
+        {
+            Console.WriteLine(@"
+                ====================================
+                BANGAZON INC CUSTOMER SERVICE PORTAL
+                ====================================");
+            Console.WriteLine("Enter the customers first and last name to start:");
+            Console.WriteLine("***********************************************************************");
+
+            CustomerFactory customerFactory = new CustomerFactory();
+            List<Customer> customerList = customerFactory.getAll();
+            foreach (Customer customer in customerList)
+            {
+                Console.WriteLine($"{customer.CustomerId} { customer.FirstName} {customer.LastName} ");
+            }
+            Console.WriteLine("***********************************************************************");
+
+            string customerName = Console.ReadLine();
+            Customer customerId = customerFactory.getCustomerByFullName(customerName);
+
+            while (customerId == null)
+            {
+                customerId = customerFactory.getCustomerByFullName(customerName);
+                if (customerId == null)
+                {
+                    Console.WriteLine("This is not a valid answer. Please try again!");
+                    customerName = Console.ReadLine();
+                }
+            }
+
+            int orderID = 0;
+            Console.WriteLine(@"
+                ====================================
+                BANGAZON INC CUSTOMER SERVICE PORTAL
+                ====================================");
+            Console.WriteLine("Choose a customer order:");
+                OrderFactory orderFactory = new OrderFactory();
+                List<Order> orderList = orderFactory.getAllOrdersFromCustomer(customerId.CustomerId);
+                foreach (Order order in orderList)
+                {
+                    Console.WriteLine($"Order {order.OrderId}: { order.Date}");
+
+                }
+            Console.WriteLine("X.Exit");
+
+            while (orderID < 0)

```

Comment:
The program never enters this `while` loop as far as I can tell. I think you want: `orderID == 0`